tcms/templates/plan/get_cases.html

```diff
@@ -16,38 +16,38 @@
 						{% if perms.testcases.add_testcase %}
-						<li><input type="button" id="js-clone-case" class="clone icon_plan" value="Clone" title="Clone selected cases to another test plan" data-param="{% url "testcases-clone" %}" /></li>
+						<li><input type="button" id="js-clone-case" class="clone icon_plan" value="{% trans "Clone" %}" title="{% trans "Clone selected cases to another test plan" %}" data-param="{% url "testcases-clone" %}" /></li>
 						{% else %}
-						<li><input type="button" class="clone icon_plan" value="Clone" title="Clone selected cases to another test plan" disabled /></li>
+						<li><input type="button" class="clone icon_plan" value="{% trans "Clone" %}" title="{% trans "Clone selected cases to another test plan" %}" disabled /></li>
 						{% endif %}
 						{% if perms.testcases.delete_testcaseplan %}
-						<li><input type="button" id="js-remove-case" class="delete icon_plan" value="Remove" title="Remove selected cases from this plan" /></li>
+						<li><input type="button" id="js-remove-case" class="delete icon_plan" value="{% trans "Remove" %}" title="{% trans "Remove selected cases from this plan" %}" /></li>
 						{% else %}
-						<li><input type="button" class="delete icon_plan" value="Remove" title="Remove selected cases from this plan" disabled /></li>
+						<li><input type="button" class="delete icon_plan" value="{% trans "Remove" %}" title="{% trans "Remove selected cases from this plan" %}" disabled /></li>
 						{% endif %}
 					</ul>
 				</li>
                         {% if REQUEST_CONTENTS.template_type == 'case' and perms.testruns.add_testrun %}
                                 <li>
                                     <span class='sprites toolbar_run{% if not perms.testruns.add_testrun %}_disabled{% endif %} icon_plan'>{% trans "Run" %}</span>
                                     <ul>
-                                        <li><input type="button" class="new_run icon_plan" value="Create new TestRun" id="js-new-run" data-param="{% url "testruns-new" %}" /></li>
+                                        <li><input type="button" class="new_run icon_plan" value="{% trans "Create new TestRun" %}" id="js-new-run" data-param="{% url "testruns-new" %}" /></li>
                                     </ul>
                                 </li>
                         {% endif %}
 			{% if perms.testcases.change_testcase %}
                                 {% if REQUEST_CONTENTS.template_type == 'review_case' %}
-                                <li><input type="button" value="Reviewer"       title="Change TestCase reviewer" class="btn_reviewer       assignee sprites icon_plan" /></li>
+                                <li><input type="button" value="Reviewer" title="Change TestCase reviewer" class="btn_reviewer assignee sprites icon_plan" /></li>
```

Comment:
Missing trans tag here. Probably you want to add it?

Reply:
Done